### ISSUE TITLE
Installation instructions in README requires an update?

### ISSUE THREAD

**Issue by abyvinod** [natchi92/stochy#1]
I am trying to install StocHy on Ubuntu 18.04 from the github sources. I am using the release version route.

These are the steps I have completed so far:
1. Clone the repository https://github.com/natchi92/stochy to my local machine
2. Run `git submodule update --init` (Needs to be mentioned in the README)
3. Run `./get_dep.dist.sh` (Completes successfully)
4. Run `./build_release.sh` (Fails)
```
-- Found PythonLibs: /workspace/Software/anaconda3/lib/libpython3.7m.so (found suitable exact version "3.7.3") 
CMake Warning at lib/nlopt/CMakeLists.txt:299 (find_package):
  By not providing "FindNumPy.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "NumPy", but
  CMake did not find one.

  Could not find a package configuration file provided by "NumPy" with any of
  the following names:

    NumPyConfig.cmake
    numpy-config.cmake

  Add the installation prefix of "NumPy" to CMAKE_PREFIX_PATH or set
  "NumPy_DIR" to a directory containing one of the above files.  If "NumPy"
  provides a separate development package or SDK, be sure it has been
  installed.


CMake Warning at lib/nlopt/CMakeLists.txt:315 (find_package):
  By not providing "FindGuile.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Guile", but
  CMake did not find one.

  Could not find a package configuration file provided by "Guile" with any of
  the following names:

    GuileConfig.cmake
    guile-config.cmake

  Add the installation prefix of "Guile" to CMAKE_PREFIX_PATH or set
  "Guile_DIR" to a directory containing one of the above files.  If "Guile"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Found SWIG: /usr/bin/swig3.0 (found version "3.0.12") 
CMake Error at lib/nlopt/src/swig/CMakeLists.txt:6 (include):
  include could not find load file:

    TargetLinkLibrariesWithDynamicLookup


CMake Warning at lib/nlopt/CMakeLists.txt:326 (find_package):
  By not providing "FindOctave.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Octave", but
  CMake did not find one.

  Could not find a package configuration file provided by "Octave" with any
  of the following names:

    OctaveConfig.cmake
    octave-config.cmake

  Add the installation prefix of "Octave" to CMAKE_PREFIX_PATH or set
  "Octave_DIR" to a directory containing one of the above files.  If "Octave"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Could NOT find Matlab (missing: Matlab_INCLUDE_DIRS Matlab_MEX_LIBRARY Matlab_MEX_EXTENSION Matlab_ROOT_DIR MX_LIBRARY MAIN_PROGRAM) (found version "NOTFOUND")
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
ARMA_LIB
    linked by target "common" in directory XXX/stochy/src/common
    linked by target "shs" in directory XXX/stochy/src/shs
    linked by target "FAUST" in directory XXX/stochy/src/FAUST
    linked by target "bmdp" in directory XXX/stochy/src/bmdp
    linked by target "task_exec" in directory XXX/stochy/src/task_exec
GINAC_LIB
    linked by target "shs" in directory XXX/stochy/src/shs
    linked by target "FAUST" in directory XXX/stochy/src/FAUST
    linked by target "bmdp" in directory XXX/stochy/src/bmdp
MATIO_LIB
    linked by target "shs" in directory XXX/stochy/src/shs

-- Configuring incomplete, errors occurred!
```

Can you tell me how to go about editing CMakeLists.txt?

**Comment by abyvinod** [comment]
Running `./get_dep.dev.sh` seems to reduces the errors related to `GINAC_LIB` and `MATIO_LIB`.
```
CMake Warning at lib/nlopt/CMakeLists.txt:299 (find_package):
  By not providing "FindNumPy.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "NumPy", but
  CMake did not find one.

  Could not find a package configuration file provided by "NumPy" with any of
  the following names:

    NumPyConfig.cmake
    numpy-config.cmake

  Add the installation prefix of "NumPy" to CMAKE_PREFIX_PATH or set
  "NumPy_DIR" to a directory containing one of the above files.  If "NumPy"
  provides a separate development package or SDK, be sure it has been
  installed.


CMake Warning at lib/nlopt/CMakeLists.txt:315 (find_package):
  By not providing "FindGuile.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Guile", but
  CMake did not find one.

  Could not find a package configuration file provided by "Guile" with any of
  the following names:

    GuileConfig.cmake
    guile-config.cmake

  Add the installation prefix of "Guile" to CMAKE_PREFIX_PATH or set
  "Guile_DIR" to a directory containing one of the above files.  If "Guile"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Found SWIG: /usr/bin/swig3.0 (found version "3.0.12") 
CMake Error at lib/nlopt/src/swig/CMakeLists.txt:6 (include):
  include could not find load file:

    TargetLinkLibrariesWithDynamicLookup


CMake Warning at lib/nlopt/CMakeLists.txt:326 (find_package):
  By not providing "FindOctave.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Octave", but
  CMake did not find one.

  Could not find a package configuration file provided by "Octave" with any
  of the following names:

    OctaveConfig.cmake
    octave-config.cmake

  Add the installation prefix of "Octave" to CMAKE_PREFIX_PATH or set
  "Octave_DIR" to a directory containing one of the above files.  If "Octave"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Could NOT find Matlab (missing: Matlab_INCLUDE_DIRS Matlab_MEX_LIBRARY Matlab_MEX_EXTENSION Matlab_ROOT_DIR MX_LIBRARY MAIN_PROGRAM) (found version "NOTFOUND")
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
ARMA_LIB
    linked by target "common" in directory XXX/stochy/src/common
    linked by target "shs" in directory XXX/stochy/src/shs
    linked by target "FAUST" in directory XXX/stochy/src/FAUST
    linked by target "bmdp" in directory XXX/stochy/src/bmdp
    linked by target "task_exec" in directory XXX/stochy/src/task_exec

-- Configuring incomplete, errors occurred!
```

**Comment by natchi92** [comment]
Can you please confirm your Gcc version and Python versions? Matplotlib had depended on python 2.7

**Comment by natchi92** [comment]
Updated fix within latest commit  4e2995b
